From dev@capelis.dj  Wed Mar 11 13:40:12 2009
From: "D.J. Capelis" <dev@capelis.dj>
Date: Wed, 04 Mar 2009 10:27:52 -0800
Subject: USB: pedantic: spelling correction in comment for ch9.h
To: Greg K-H <gregkh@suse.de>
Message-ID: <1236191272.19965.51.camel@aes.capelis.dj>


Just noticed this during a grep, figured I might as well send it in.

From: D.J. Capelis <dev@capelis.dj>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/usb/ch9.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/linux/usb/ch9.h
+++ b/include/linux/usb/ch9.h
@@ -102,7 +102,7 @@
 #define USB_REQ_LOOPBACK_DATA_READ	0x16
 #define USB_REQ_SET_INTERFACE_DS	0x17
 
-/* The Link Power Mangement (LPM) ECN defines USB_REQ_TEST_AND_SET command,
+/* The Link Power Management (LPM) ECN defines USB_REQ_TEST_AND_SET command,
  * used by hubs to put ports into a new L1 suspend state, except that it
  * forgot to define its number ...
  */
